### DISCUSSION TITLE
How can i connect to different network on Phantom using deeplinks??

### DISCUSSION THREAD

**Discussion by cheetoss37** [phantom/docs#318] [Help & Support] [answered]
### Summary

I’ve success connect my dapp to phantom using deeplinks on Solana

But how can i connect to other chains using deeplinks, Ethereum for example?? 

Since the “cluster” parameter when execute method “connect” on the documentation only show for Solana

### Steps to Reproduce

N/A

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
@cheetoss37 For the time being deeplinks are only supported for the Solana network

**Reply by cheetoss37** [reply]
@adamdelphantom Noted, thank you for the response
